data/json/recipes/recipe_electronics.json

```diff
@@ -2370,7 +2374,10 @@
   "components": [
     [ [ "processor", 1 ] ],
     [ [ "scrap", 2 ] ],
-    [ [ "wire", 5 ] ],
+    [
```

Comment:
Incorrect. Copper wire is item ID `cable`, `wire` is generic metal fence wire. One `wire` is 151 grams, 5 of them are 755 grams. I could have made the recipe 4 wire or 1 ingot (604 grams versus 675) instead.